nsv13/code/game/objects/items/devices/nsv13_megaphone.dm

```diff
@@ -0,0 +1,4 @@
+//A special megaphone for fans of Admiral Rudiger
```

Comment:
This does not need an entire file just for itself.
Use nsv13/code/game/objectives/items/custom_items.dm